### ISSUE TITLE
Consider changing 'dependency' decorator to 'tailor_dependency'

### ISSUE THREAD

**Issue by gpennington** [divisible-by-hero/tailor#15]
Namespacing really.

Also, consider adding both 'tailored' tasks and 'dependency' tasks top the schema task list BUT give the task a 'availabe' property or something.  So, they are in the same list but dependencies aren't directly callable.

**Comment by dstegelman** [comment]
True, however they are already sorta name spaced based upon the import.  I see what you are saying though.
